src/Og.php

```diff
@@ -117,7 +116,10 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
    *  the OG membership ID and the group ID as the value. If nothing found,
    *  then an empty array.
    */
-  public static function getEntityGroups($entity_type, $entity_id, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+  public static function getEntityGroups(EntityInterface $entity, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+    $entity_type = $entity->getEntityTypeId();
```

Comment:
> I don't mind too much

Me neither, just want to be consistent. I could go with `$entity_type_id` as-well.